### ISSUE TITLE
Add Coin Market Scanner Signals

### ISSUE THREAD

**Issue by talvasconcelos** [carlos8f/zenbot#1862]
Guys don't know if devs are interested or not but, i did an AI signals website that works for binance. At the moment it sends signals to the website. As I'm trying to build a simple trader bot to use them, i spinned a websocket that pushes the signals for the trader to run. 

@deviavir or any devs here want to integrate the signals on zenbot? it's not the normal workflow of zenbot as it would trade signals instead of strategy based entries...

https://coinmarketscanner.app 

if this is inappropriate, please let me know and I'll delete the post/issue.

Best regards,
Tiago Vasconcelos

**Comment by hyuce** [comment]
First of all, congratulations. What did you encode the interface with? Is it Open source? What kind of algorithm did you use? Can we make a contribution?

**Comment by talvasconcelos** [comment]
hi, thanks for your reply. I'm using keras for training... i'm not a python coder, so my code is a mess!!

i prepare data on json (on observable) then upload to google colab to train on python and export the model to tensorflow js so i can do inference on the browser. the code is open source, you can take a look at the colab notebook: https://colab.research.google.com/drive/1JKN62U-XTAug7RkGhg1qrVyjuFQYOc6E

and the data prep observable: https://observablehq.com/@talvasconcelos/export-data-to-colab

ii'm constantly trying to make a better model, and better data prep to feed the model, so this is not final, or closed code!

the signal come from the server, check out the code at: https://github.com/talvasconcelos/scanner_server

**Comment by talvasconcelos** [comment]
I can possibly add a strategy to use the signals, saw recently that zenbot uses ws!
it would only take advantage of the config options... sell price, trailing stop, stop loss!!

**Comment by hyuce** [comment]
Hi;
I'm trying to understanding out the code :)
Really, it's complicated. I think if you place it in a github archive it's less complicated  and I can give you contribute.

> hi, thanks for your reply. I'm using keras for training... i'm not a python coder, so my code is a mess!!
> 
> i prepare data on json (on observable) then upload to google colab to train on python and export the model to tensorflow js so i can do inference on the browser. the code is open source, you can take a look at the colab notebook: https://colab.research.google.com/drive/1JKN62U-XTAug7RkGhg1qrVyjuFQYOc6E
> 
> and the data prep observable: https://observablehq.com/@talvasconcelos/export-data-to-colab
> 
> ii'm constantly trying to make a better model, and better data prep to feed the model, so this is not final, or closed code!
> 
> the signal come from the server, check out the code at: https://github.com/talvasconcelos/scanner_server

**Comment by hyuce** [comment]
Of course. But if you want it to contribute, you should use a github archive as I said.

> I can possibly add a strategy to use the signals, saw recently that zenbot uses ws!
> it would only take advantage of the config options... sell price, trailing stop, stop loss!!

**Comment by talvasconcelos** [comment]
Hi, as soon as i can get to it i'll try to clean up the code. Python or JS you need? 
Also, what do you want to do? add the model to be trained by zenbot users?
